content/en/event-streaming.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Event Streaming
+status: Feedback Appreciated
+category: concept
+---
+
+## What it is
+
+Event streaming is an approach where software sends event data from one application to another to continuously communicate what they are doing.
+Picture a service broadcasting everything it does to all other services.
+Each activity taken by a service is referred to as an event, hence event streaming.
+For example, the NASDAQ gets updates on stock and commodities pricing every second. If you had an application that monitored a specific set of stocks, you would want to receive that information in near real-time. Yahoo! Finance provides an [API](https://glossary.cncf.io/application-programming-interface/) that pulls from NASDAQ and sends (or streams) the information (or events) from their application to any application that subscribes to it.
+The data being sent as well as the changes in that data (stock prices) are the events while the process of delivering them to an application is event streaming.
+
+## Problem it addresses
+
+Traditionally, Yahoo! Finance would use single TCP requests. This would be very inefficient as it would require a connection to be created for every event. As data becomes more real-time in nature, scaling such a solution would be inefficient.
```

Comment:
```suggestion
For example, the NASDAQ gets updates on stock and commodities pricing every second.
If you had an application that monitored a specific set of stocks, you would want to receive that information in near real-time.
Yahoo! Finance provides an [API](https://glossary.cncf.io/application-programming-interface/) that pulls from NASDAQ and sends (or streams) the information (or events) from their application to any application that subscribes to it.
The data being sent as well as the changes in that data (stock prices) are the events while the process of delivering them to an application is event streaming.

## Problem it addresses

Traditionally, Yahoo! Finance would use single TCP requests.
This would be very inefficient as it would require a connection to be created for every event.
As data becomes more real-time in nature, scaling such a solution would be inefficient.
```

Thank you @jasonsmithio . I love the collaborations in this PR to improve quality!

The recent change does not follow the style-guide (https://glossary.cncf.io/style-guide/#three-subheadings). So, I'd like to suggest line breaks. :)
(Please note that I didn't review the contents itself)

> To facilitate review, please use semantic line breaks (one sentence per line).